lib/include/elements/view.hpp

```diff
@@ -102,8 +102,13 @@ namespace cycfi { namespace elements
       using io_context = asio::io_context;
       io_context&             io();
 
+      struct timer
```

Comment:
Perhaps `timer` is not the best name for this? Ah, unless you want to add more timer-related functions later?

Reply:
Yeah that was my feeling. But I start to think even if we add more timer functionalities (maybe for an animation system?) the right place for them won't be there.

I can rename it to sth more contextual, like `timer_canceller`? Or `callback_canceller`?

Reply:
How about just returning the timer?

Reply:
I can do that too. I don't know if things can break if the developer use other methods than `cancel`, but I doubt it will. I'll try that

Reply:
Asio is stable. It's been in Boost for a VERY long time now. It was actually considered to be in std at one point. It's up to the developer to know and use the asio timer API. But it's quite simple to use.